src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
+
+
+## Step 1: Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry data](self-hosted/helm-configuration.mdx#telemetry) to help us understand usage patterns and prioritize improvements
+In air-gapped environments, you can disable telemetry by adding the following to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+Disabling telemetry will automatically stop Okteto CLI analytics for all developers.
+
+## Step 2: Host Images in your Private Registry
+
+By default, all the images needed to deploy the Okteto Platform, and the images used by the Okteto CLI, are hosted in DockerHub.
+In air-gapped environments, you’ll need to host these images in your private registry.
+
+### Push Images to your Private Registry
+
+For simplicity, we recommend using the same repository names and tags in your private registry.
+
+:::info
+Okteto doesn't support `imagePullSecrets`, so your cluster must be configured to have access to your Private Registry, 
+:::
+
+**Required Images for Okteto Platform Installation**
+
+Push the following images to your private registry for the Okteto Chart installation:
+
+- **okteto/backend:{variables.chartVersion}**
+- **okteto/frontend:{variables.chartVersion}**
+- **okteto/buildkit:{variables.chartVersion}**
+- **okteto/registry:{variables.chartVersion}**
+- **okteto/pipeline-installer:{variables.chartVersion}**
+- **okteto/pipeline-runner:{variables.chartVersion}**
+- **okteto/ingress-nginx:{variables.chartVersion}**
+- **okteto/reloader:{variables.chartVersion}**
+- **okteto/redis:{variables.chartVersion}**
+
+For each Okteto CLI version, push the following images to your private registry:
+
+- **okteto/okteto:{variables.cliVersion}**
+- **okteto/dockerfile:{variables.cliVersion}**
+
+:::info
+For Okteto Chart {variables.chartVersion}, we recommend enforcing the usage of Okteto CLI {variables.cliVersion} in your developer's machines to reduce maintenance overhead
```

Comment:
can we include a link on how to do this? 

Also, we shouldn't recommend, we should tell them this is  a must, otherwise things won't work (e.g. if they let developers install liberally, the okteto/okteto:VERSION image won't be available on the cluster and it will fail. I think 'setting the supported CLI' versions should be a step on the guide. 

In general, with air gapped we need to give stronger direction than on public cloud because every small deviation/variation can break things and slow down the installation.